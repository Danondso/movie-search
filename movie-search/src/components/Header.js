import React from 'react';

class Header extends React.Component {

    test() {
        return "hi";
    };
    render() {
        return (
            <div>
            <input type="text" placeholder="movie-title"></input>
            </div>
        );
    }    
}

export default Header;