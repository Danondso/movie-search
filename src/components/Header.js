import React from "react";
import { makeStyles } from "@material-ui/core/styles";

const useStyles = makeStyles(theme => ({
  header: {
    backgroundColor: "#282c34",
    minHeight: "10vh",
    alignItems: "center",
    justifyContent: "center",
    fontSize: "3rem",
    color: "white",
    display: "flex"
  }
}));

const Header = props => {
  const classes = useStyles();
  return <header className={classes.header}>{props.title}</header>;
};

export default Header;
