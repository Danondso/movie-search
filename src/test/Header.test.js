import React from 'react';
import Header from '../components/Header';
import renderer from 'react-test-renderer';

test('Search renders', () => {
    const component = renderer.create(
        <Header />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});