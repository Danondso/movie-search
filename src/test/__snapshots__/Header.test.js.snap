// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search renders 1`] = `
<header
  className="makeStyles-header-1"
/>
`;
